plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -122,9 +122,17 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
     const { nodeHealthResponse, filteredNodes, masterRegion } = this.state;
     const { networks = [] } = this.props;
 
+    // Detecting whether DCOS already supports maintenance mode.
+    // We might want to remove this flag at some point in the future.
+    const hasMaintenanceMode = CompositeState.getNodesList().list.some(
```

Comment:
I'm wondering if we should centralize these types of conditions and use a context and context consumer to get these flags from anywhere in the hierarchy 🤔

Reply:
i'm not sure. i'd rather have a static feature flag or a feature-flag-module that allows for runtime-toggling. 🤔 

i think there's nothing too bad here. having to look through the list naively admittedly feels bad although it should terminate on the first entry for DCOS'es having that feature.

good enough for now?